private/protocol/jsonrpc/unmarshal_error.go

```diff
@@ -22,19 +22,21 @@ const (
 // UnmarshalTypedError provides unmarshaling errors API response errors
 // for both typed and untyped errors.
 type UnmarshalTypedError struct {
-	exceptions map[string]func(protocol.ResponseMetadata) error
+	exceptions      map[string]func(protocol.ResponseMetadata) error
```

Comment:
nit: were these extra spaces/tab intentional?

Reply:
discussed offline. resolving this thread because these formatting changes are intended and correct